docs/content/setup/debian.md

```diff
@@ -59,6 +59,6 @@ To check the status of the Monkey Island after the installation, run the followi
 
 To upgrade when a new version comes out, download the new Monkey `.deb` file and install it. You should see a message like `Unpacking monkey-island (1.8.2) over (1.8.0)`. After which, the installation should complete successfully.
 
-If you'd like to keep your existing configuration, you can export it to a file by using the Export button and then import it to the new server.
+If you'd like to keep your existing configuration, you can export it to a file using the export button and then import it to the new server.
```

Comment:
Waiting on a resolution to https://github.com/guardicore/monkey/pull/934/files#r562772695